core/domain/skill_services.py

```diff
@@ -232,156 +232,161 @@ def get_all_skill_summaries():
 
     Returns:
-        4-tuple(skill_summaries, next_cursor, more, total_count). where:
-            skill_summaries: list(AugmentedSkillSummary). The list of
-                skill summaries.
-            next_cursor: str or None. A query cursor pointing to the
-                next batch of results. If there are no more results,
-                 this might be None.
+        3-tuple(skill_summaries, next_cursor, more). where:
+            skill_summaries: list(SkillSummary). The list of skill summaries.
+            urlsafe_start_cursor: str or None. A query cursor pointing to the
+                next batch of results. If there are no more results, this might
+                be None.
             more: bool. If True, there are (probably) more results after this
                 batch. If False, there are no further results after this batch.
-            total_count: number. Total count of skill summaries.
     """
-    skill_summaries_models, next_cursor, more, total_count = (
+    skill_summaries_models, urlsafe_start_cursor, more = (
         skill_models.SkillSummaryModel.fetch_page(
-            num_queries_to_fetch, cursor, sort_by))
+            2 * num_items_to_fetch, cursor, sort_by))
 
     skill_summaries = [
         get_skill_summary_from_model(summary)
         for summary in skill_summaries_models]
-    return skill_summaries, next_cursor, more, total_count
+    return skill_summaries, urlsafe_start_cursor, more
 
 
 def get_filtered_skill_summaries(
-        num_queries_to_fetch, status, classroom_name, keywords,
+        num_items_to_fetch, status, classroom_name, keywords,
         sort_by, cursor):
     """Returns all the skill summary dicts after filtering.
 
     Args:
-        num_queries_to_fetch: number. Number of skills to fetch
+        num_items_to_fetch: int. Number of skills to fetch.
         status: str. The status of the skill.
-        classroom_name: string. The classroom_name of the topic
-            to which skill is assigned to.
-        keywords: list(string). The keywords to look for
+        classroom_name: str. The classroom_name of the topic to which skill is
+            assigned to.
+        keywords: list(str). The keywords to look for
             in the skill description.
-        sort_by: string. A string indicating how to sort the result.
+        sort_by: str. A string indicating how to sort the result.
         cursor: str or None. The cursor to the next page.
 
     Returns:
-        4-tuple(query_models, next_cursor, more, count). where:
-            skill_summaries: list(AugmentedSkillSummary). The list of
-                skill summaries.
-            next_cursor: str or None. A query cursor pointing to the
-                next batch of results. If there are no
-                more results, this might be None.
+        3-tuple(query_models, next_cursor, more, count). where:
+            augmented_skill_summaries: list(AugmentedSkillSummary). The list of
```

Comment:
You have not used augmented_skill_summaries in your description of the 3-tuple.

Your 3-tuple above seems to be a 4-tuple.

Reply:
Oops! Fixed.